### DISCUSSION TITLE
Elevation Data for JSOW's on Kneeboard

### DISCUSSION THREAD

**Discussion by Kettenkoeter** [Khopa/dcs_liberation#1825] [Q&A] [answered]
Elevation Data for JSOW's on Kneeboard is missing.
Would be nice if we could have that data on the kneeboard.
There's only target coordinates and the ALIC code.

**Comment by Starfire13** [comment, marked as answer]
Just slave TGP to waypoint and drop the JSOW. You don't need elevation data because the waypoint is already set to ground level. Besides, Liberation doesn't have access to elevation data to generate it onto the kneeboard.

**Reply by Kettenkoeter** [reply]
Thanks for the info

**Comment by Simba75013** [comment]
The problem is AGL is not useful for long release, like JSOWs or SLAMER, or lofting JDAMs.

The best way to have the MSL is to go into observer mode at mission start and note coordinates and elevation.
